eventmesh-sdk-java/build.gradle

```diff
@@ -15,10 +15,15 @@
  * limitations under the License.
  */
 
+plugins {
+    id 'java-library'
+    id 'com.github.johnrengelman.shadow' version '7.1.2'
```

Comment:
After I use the `java-library` and change the dependency configuration to `api`, the built jar file still does not contain the dependencies.
Thus the `shadow` plugin used here to generate a fat jar. (for your [reference](https://discuss.gradle.org/t/how-to-include-dependencies-in-jar/19571))